Dockerfile

```diff
@@ -0,0 +1,166 @@
+ARG unitycatalog_uid=185
+ARG unitycatalog_home="/opt/unitycatalog"
+ARG unitycatalog_repo="${unitycatalog_home}/repo"
+ARG unitycatalog_jar="server/target"
+ARG unitycatalog_etc="etc"
+ARG unitycatalog_bin="bin"
+ARG unitycatalog_user_name="unitycatalog"
+ARG unitycatalog_user_home="home"
+ARG unitycatalog_user_basedir="${unitycatalog_home}/${unitycatalog_user_home}"
+# Specify any custom parameters necessary to generate
+# the Uber-Jar by SBT. 
+# Note: The default allocated heam memory size is too small
+# and will cause the process to fail when attempting to compile
+# and generate the Uber-Jar. Therefore it is important to choose
+# a size large enough for the compiler to run. 
+ARG sbt_args="-J-Xmx5G"
+
+
+
+# ###### STAGE 1 ###### #
+# Building the Uber-Jar #
+#########################
+
+FROM eclipse-temurin:22-jdk-alpine AS build_uc
+
+ARG unitycatalog_repo
+ARG sbt_args
+
+# Install any required packages
+RUN <<EOF
+    set -ex;
+    apk update;
+    apk upgrade; 
+    apk add bash git;
+    rm -R /var/cache/apk/*;
+EOF
+
+# Create temporary directories for initialization files
+RUN mkdir -p "${unitycatalog_repo}";
+
+WORKDIR "${unitycatalog_repo}"
+
+COPY . .
+
+RUN build/sbt ${sbt_args} server/assembly
+
+
+# ###### STAGE 2 ###### #
+# Running the UC server #
+#########################
+
+FROM eclipse-temurin:22-jre-alpine AS run_uc
+
+ARG unitycatalog_uid
+ARG unitycatalog_home
+ARG unitycatalog_repo
+ARG unitycatalog_jar
+ARG unitycatalog_etc
+ARG unitycatalog_bin
+ARG unitycatalog_user_name
+ARG unitycatalog_user_home
+ARG unitycatalog_user_basedir
+ARG sbt_args
+
+EXPOSE 8081
```

Comment:
Saw your comment on port + 1, but the default port is still 8080. We start the server on 8081 but there's a transcoder verticle on 8080 that forwards requests to 8081. So we would still want to hit 8080 for requests. Look [here](https://github.com/unitycatalog/unitycatalog/blob/main/server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java#L94).

Reply:
Agree, should expose 8080, and probably only that port.

Reply:
Yeah but I'm still not convinced that I should.
I need to test again with 8080 and if it works I'll make the change.